app/factories/bulkrax/valkyrie_object_factory.rb

```diff
@@ -155,7 +155,7 @@ def self.find(id)
       Hyrax.query_service.find_by(id: id)
       # Because Hyrax is not a hard dependency, we need to transform the Hyrax exception into a
```

Comment:
this method previously didn't work with valkyrization. Now that it is working, we needed to capture the valkyrie errors as well.